src/main/resources/assets/api-docs.yaml

```diff
@@ -275,14 +275,26 @@ paths:
           description: Internal Server Error
   /api/dacuser:
     $ref: './paths/dacuser.yaml'
+
   /api/dar/v2:
     $ref: './paths/darV2.yaml'
+
+  /api/dar/v3:
+    $ref: './paths/darV3.yaml'
+
   /api/dar/v2/{referenceId}:
     $ref: './paths/darV2ByReferenceId.yaml'
+
+  /api/dar/v3/{referenceId}:
+    $ref: './paths/darV3ByReferenceId.yaml'
+
   /api/dar/v2/draft:
     post:
+      deprecated: true
```

Comment:
Did we want to deprecate this now and not wait for the cleanup to completely switch over to the endpoints that enforce DAAs?